jinja_utils.py

```diff
@@ -72,7 +65,6 @@ def get_complete_static_resource_url(domain_url, resource_suffix):
         return '%s%s%s' % (
             domain_url, utils.get_asset_dir_prefix(), resource_suffix)
 
-    env.globals['get_static_resource_url'] = get_static_resource_url
     env.globals['get_complete_static_resource_url'] = (
```

Comment:
Is this one still needed?

Reply:
This is still used in [base.html:46](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/base.html#L46). It should be replaced too, I will see if I would have time to include it in this PR, the priority is to merge the PR before I go away.